cypress/features/regression/buttonToggleGroup.feature

```diff
@@ -34,27 +34,30 @@ Feature: Button Toggle Group component
   # | <>                       |
 
   @ignore
-  # ignored because the width renders into px
+  # @ignore until solution is applied to round sizes as integers as Chrome v80 has amended pixel sizes
   Scenario Outline: Change Button Toggle Group input width to <width>
-    When I check labelInline checkbox
-      And I set inputWidth to "<width>"
+    Given I check labelInline checkbox
+    When I set inputWidth to "<width>"
     Then input width is set to "<px>"
     Examples:
-      | width | px  |
-      | 1     | 11  |
-      | 10    | 83  |
-      | 100   | 304 |
+      | width | px        |
+      | 1     | 10.75     |
+      | 10    | 107.59375 |
+      | 100   | 825.71875 |
 
   @negative
   Scenario Outline: Set out of scope characters to Button Toggle Group input width to <width>
-    When I check labelInline checkbox
-      And I set inputWidth to "<width>"
+    Given I check labelInline checkbox
+    When I set inputWidth to "<width>"
```

Comment:
Regarding the `@ignore`, as for buttonToggle.feature, it used to work with the long decimals specified in the `Examples` and some integers until Chrome updated to v80. Now none of them work. All the dimensions are now incorrect. Rather than fail every run, it may as well be ignored until we can make it less brittle. I previously added a comment about why it is ignored stating "@ignore until solution is applied to round sizes as integers as Chrome v80 has amended pixel sizes".

In terms of the inputWidth issue, this scenario and `Examples` table were already in-situ. I can specify just one width in this step if that's all that is required.

Reply:
Related to `Examples` - ye, please use just only one width. Then you won't need `Examples` anymore

Reply:
@KatarzynaQA Does the same apply to the similar scenario for labelWidth?
`Scenario Outline: Set out of scope characters to Button Toggle Group label width to <width>`
    `Given I check labelInline checkbox`
    `When I set labelWidth to "<width>"`
    `Then label width is not set "<width>"`
    `Examples:`
      `| width                |`
      `| Sample text          |`
      `| áéíóú¿¡üñ            |`
      `| !@#$%^*()_=~[];:,?{} |`
      `| ÄÖÜßäöüß             |`
      `| <>                   |`

Reply:
Removed Examples and added a multi-character string as the "width" as the test scenario was to test out of scope characters.